src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -910,6 +918,8 @@ void consumerAlreadyExistsTest(VertxTestContext context) throws InterruptedExcep
                     create3Again.complete(true);
                 });
 
+        consumerService().deleteConsumer(context, groupId, name);
+        consumerService().deleteConsumer(context, groupId, name + "diff");
```

Comment:
you should first wait for `create3Again` to be set and then you can deleting consumers.
Again, delete method on next line :-)